labelbox/schema/model_run.py

```diff
@@ -456,13 +456,16 @@ def export_labels(
     
     """
 
-    def export_v2(self, task_name: str,
-                  params: Optional[ModelRunExportParams]) -> Task:
-        _params = params or {}
+    def export_v2(self,
+                  task_name: Optional[str] = None,
+                  params: Optional[ModelRunExportParams] = {}) -> Task:
```

Comment:
Assigning a default argument of a dict is bad practice. It isn't re-defined for each function invokation, only when the function is defined.

https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python 

Technically you aren't modifying this so it should be fine. Also, optional means default value of None - not that it has a default. 

I would revert back to how you have it. But just set the value to None by default.